nodes/src/nodes/agent_deepagent/IInstance.py

```
@@ -77,4 +79,39 @@ def invoke(self, param: Any) -> Any:  # noqa: ANN401
         op = param.get('op') if isinstance(param, dict) else getattr(param, 'op', None)
         if isinstance(op, str) and op.startswith('tool.'):
             return self.IGlobal.agent.handle_invoke(self, param)
+        if isinstance(op, str) and op == 'deepagent.describe':
+            self._handle_deepagent_describe(param)
+            return
```

Ruff (0.15.9)

[warning] Do not implicitly `return None` in function able to return non-`None` value

Add explicit `None` return value

(RET502)